meteorite_fall.js

```diff
@@ -49,6 +54,25 @@ function initializeMeteorites() {
     }
 }
 
+function checkGameOver(meteoriteX, meteoriteY, playerX, playerY, meteoriteWidth, meteoriteHeight, playerWidth) {
+    if (meteoriteY + meteoriteHeight == playerY) {
+        if (meteoriteX <= playerX + playerWidth && meteoriteX + meteoriteWidth >= playerX) {
```

Comment:
why not combine these two if statements into one using an && operator?

Reply:
From testing it seems that the game ends the frame before the actual collision, not when u actually collide. I think this may be because of the meteoriteY + meteoriteHeight